### ISSUE TITLE
make new image repos public

### ISSUE THREAD

**Issue by joshgav** [podtato-head/podtato-head#138]
**TL;DR** - we need to make the image repos at `ghcr.io/podtato-head/{entry,hat,right-leg,left-leg,right-arm,left-arm}` public, and that requires an org admin either here in [github.com/podtato-head](https://github.com/orgs/podtato-head/packages), or possibly still in [github.com/cncf](https://github.com/orgs/cncf/packages).

@thschue or @AloisReitbauer can you please take a look at the private packages for those orgs (you can click the links above) and make the relevant ones public? Thanks!

### Background

The container image repos associated with this git repo - that is, those at `ghcr.io/podtato-head/podtato-head/{entry,hat,right-leg,left-leg,right-arm,left-arm}` - are public and can be viewed at <https://github.com/orgs/podtato-head/packages?repo_name=podtato-head> and pulled with `docker pull ghcr.io/podtato-head/podtato-head/entry:latest`. Admins of this repo can manage those images.

However, the container image repos where we push the _final_ builds in [the release workflow](https://github.com/podtato-head/podtato-head/blob/main/.github/workflows/release.yaml) - `ghcr.io/podtato-head/{entry,hat,right-leg,left-leg,right-arm,left-arm}` - are _not_ public. I'm not even sure if that repo is associated with the podtato-head org - it may still be associated with [github.com/cncf](https://github.com/orgs/cncf/packages). As a corollary, it isn't possible to run `docker pull ghcr.io/podtato-head/entry:latest`.

To reiterate for clarity:
- `ghcr.io/podtato-head/podtato-head/entry` is public and usable now :+1: 
- `ghcr.io/podtato-head/entry` is not yet public and can only be used in release tests :disappointed: 

The main tutorials and manifests use the short name, so they don't work at the moment.

We need to make the short path repos public, which is a setting on the package's page in GitHub. We'll know this is done when `docker pull ghcr.io/podtato-head/entry:latest` works without authentication.

**Comment by joshgav** [comment]
This is done, see <https://github.com/orgs/podtato-head/packages?repo_name=podtato-head>. Thanks @thschue for your help!
